docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -0,0 +1,219 @@
+---
+description: Learn how to get started with Grafana Tempo using the Helm chart.
+menuTitle: Get started
+title: Get started with Grafana Tempo using the Helm chart
+weight: 20
+keywords:
+  - Helm chart
+  - Kubernetes
+  - Grafana Tempo
+---
+
+# Get started with Grafana Tempo using the Helm chart
+
+The Grafana Tempo Helm chart allows you to configure, install, and upgrade Grafana Tempo within a Kubernetes cluster. Using this procedure, you will:
+
+- Create a custom name-space within your Kubernetes cluster
```

Comment:
I think just `namespace`, as it is referred to in kubernetes.

Reply:
Done